src/App.js

```diff
@@ -32,6 +35,8 @@ export default function App() {
 
   return (
     <div className="App">
+      <Organization />
```

Comment:
Great start! These components look good and behave well!

Can you update this PR to be in sync with the main branch? It looks like there's a conflict on this file and you can see what has changed. 

I would just keep all of the main branch changes and discard your own, then add these components to the new InputForm.jsx component.